SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs

```diff
@@ -18,7 +18,7 @@ public class MpAddressDto
         [JsonProperty(PropertyName = "City")]
         public string City { get; set; }
 
-        [JsonProperty(PropertyName = "State")]
+        [JsonProperty(PropertyName = "State/Region")]
```

Comment:
You might want to escape this as [State/Region]. SQL/MP will blow up on something with a hyphen in the name, not sure of side effects with this.